### PR TITLE
Update to Vert.x 3.9.0 RMB-619

### PR DESCRIPTION
**Pull request by adamdickmeiss**
Needs a little more work in "fake" connections.. so that execute fails with error (rather than query method returning null).

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -66,7 +66,7 @@
       <dependency>
         <groupId>io.vertx</groupId>
         <artifactId>vertx-stack-depchain</artifactId>
-        <version>3.8.5</version>
+        <version>3.9.0</version>
```
Comment (julianladisch) [inline pom.xml:69]: Can you also update the vertx version in sample/pom.xml and sample2/pom.xml?
Can I remove the other RMB-619 branch mentioned on Slack and in the Development section on https://issues.folio.org/browse/RMB-619?

File: domain-models-runtime/src/test/java/org/folio/rest/persist/PostgresClientTest.java [hunk 2]
```diff
@@ -321,73 +325,21 @@ public boolean isSSL() {
...
 
     @Override
-    public <R> PgConnection preparedBatch(String s, List<Tuple> list, Collector<Row, ?, R> collector, Handler<AsyncResult<SqlResult<R>>> handler) {
-      handler.handle(Future.failedFuture("not implemented"));
-      return this;
+    public void close() {
     }
   }
 
+  @Ignore
```
Comment (julianladisch) [inline domain-models-runtime/src/test/java/org/folio/rest/persist/PostgresClientTest.java:342]: Can you explain the ignores?
For example `@Ignore("Needs more work to make the fake connection work again")`

Comment (adamdickmeiss) [reply domain-models-runtime/src/test/java/org/folio/rest/persist/PostgresClientTest.java]: Back now (not ignored)

File: pom.xml (hunk 1 above)
Comment (adamdickmeiss) [reply pom.xml:69]: Updated, sample and sample2.. Funny, mvn versions:set  does not set it (it was also wrong for the 29.5.0-SNAPSHOT, where sample, sample2 was pointing to an older version).

Perhaps it's time to retire sample and sample2?

I'm confused what branch you are talking about.

Comment (julianladisch) [reply pom.xml:69]: The pom.xml of sample and sample2 contains `<vertx-version>3.8.5</vertx-version>`, this should be updated.

Comment (julianladisch) [reply pom.xml:69]: sample can be removed.
sample2 servers as a complete integration test. If we can integrate this into domain-models-runtime we can remove sample2 as well.

Comment (julianladisch) [reply pom.xml:69]: I talk about this branch: https://github.com/folio-org/raml-module-builder/compare/RMB-619-vert.x-3.9.0

File: domain-models-runtime/src/test/java/org/folio/rest/persist/PostgresClientTest.java (hunk 2 above)
Comment (julianladisch) [reply domain-models-runtime/src/test/java/org/folio/rest/persist/PostgresClientTest.java]: Very good!

File: pom.xml (hunk 1 above)
Comment (adamdickmeiss) [reply pom.xml:69]: Done

Comment (adamdickmeiss) [reply pom.xml:69]: OK. It took a while to understand. Need a break. So we both did the same work?

**Pull request merged**